Dockerfile

```
@@ -1,67 +1,83 @@
-### FIXME according to the target version
-# base image, ARG is overridable by --build-arg
-ARG BASE_IMAGE=hexpm/elixir:1.13.4-erlang-25.0.3-ubuntu-focal-20211006
+# Setup the target version
+## base image, ARG is overridable by --build-arg
+ARG BASE_IMAGE=hexpm/elixir:1.15.5-erlang-26.0.2-ubuntu-jammy-20230126
```

```suggestion
ARG BASE_IMAGE=hexpm/elixir:1.15.7-erlang-26.2.3-ubuntu-jammy-20240125
```

もしかしたら、 [bump latest branch](https://github.com/rclex/rclex_docker/compare/main...bump_latest) と同じ version にしたい？（ニュートラルな提案